launch.py

```diff
@@ -37,6 +40,17 @@ def env_defined(key):
     steamcmd.extend(["validate", "+quit"])
     subprocess.call(steamcmd)
 
+
+# move any FILES to root dir like .dll
+if os.path.exists(BASE_DIR / "move_to_root"):
+    for file in os.listdir(BASE_DIR / "move_to_root"):
+        # Check if the path is a directory
+        if os.path.isdir(BASE_DIR / "move_to_root" /file):
+            print('f{file} is a directory, create a volume link instead')
+        else:
+            # Copy the file and overwrite if it already exists
+            shutil.copy2(BASE_DIR / "move_to_root" / file, BASE_DIR / file)    
```

Comment:
Linter wants 
/github/workspace/launch.py:48:53: E225 missing whitespace around operator
/github/workspace/launch.py:52:76: W291 trailing whitespace
```suggestion
            shutil.copy2(BASE_DIR / "move_to_root" / file, BASE_DIR / file)
```